lib/CL/devices/cuda/pocl-cuda.c

```diff
@@ -756,7 +756,11 @@ pocl_cuda_init (unsigned j, cl_device_id dev, const char *parameters)
   /* Create context */
   if (ret != CL_INVALID_DEVICE)
     {
+#if CUDA_VERSION_MAJOR < 13
```

Comment:
There is a standard macro `CUDA_VERSION` defined in `cuda.h`: its value for version 13 is `13000`.